src/cable/redis_pinger.cr

```diff
@@ -9,6 +9,7 @@ module Cable
         @server.backend.ping_redis_subscribe
         @server.backend.ping_redis_publish
       rescue e
+        next stop
```

Comment:
exit the block and call `stop` to ensure the tasker is cleaned up

Reply:
The `next` keyword would exit the block early and not execute the rest of the code in the `rescue` block:
```suggestion
        stop
```